Engine/Results/RegressionResultHandler.cs

```diff
@@ -167,6 +188,124 @@ public override void SetAlphaRuntimeStatistics(AlphaRuntimeStatistics statistics
+
+            var sorted = types.Select(type => type.ToString()).OrderBy(type => type);
+            WriteLine($"SecurityTypes: {string.Join("|", sorted)}");
+        }
+
+        /// <summary>
+        /// Send a debug message back to the browser console.
+        /// </summary>
+        /// <param name="message">Message we'd like shown in console.</param>
+        public override void DebugMessage(string message)
+        {
+            base.DebugMessage(message);
+
+            WriteLine($"DebugMessage: {message}");
+        }
+
+        /// <summary>
+        /// Send an error message back to the browser highlighted in red with a stacktrace.
+        /// </summary>
+        /// <param name="message">Error message we'd like shown in console.</param>
+        /// <param name="stacktrace">Stacktrace information string</param>
+        public override void ErrorMessage(string message, string stacktrace = "")
+        {
+            base.ErrorMessage(message, stacktrace);
+
+            stacktrace = string.IsNullOrEmpty(stacktrace) ? null : Environment.NewLine + stacktrace;
+            WriteLine($"ErrorMessage: {message}{stacktrace}");
+        }
+
+        /// <summary>
+        /// Send a logging message to the log list for storage.
+        /// </summary>
+        /// <param name="message">Message we'd in the log.</param>
+        public override void LogMessage(string message)
+        {
+            base.LogMessage(message);
+
+            WriteLine($"LogMessage: {message}");
+        }
+
+        /// <summary>
+        /// Send a runtime error message back to the browser highlighted with in red
+        /// </summary>
+        /// <param name="message">Error message.</param>
+        /// <param name="stacktrace">Stacktrace information string</param>
+        public override void RuntimeError(string message, string stacktrace = "")
+        {
+            base.RuntimeError(message, stacktrace);
+
+            stacktrace = string.IsNullOrEmpty(stacktrace) ? null : Environment.NewLine + stacktrace;
+            WriteLine($"RuntimeError: {message}{stacktrace}");
+        }
+
+        /// <summary>
+        /// Send a system debug message back to the browser console.
+        /// </summary>
+        /// <param name="message">Message we'd like shown in console.</param>
+        public override void SystemDebugMessage(string message)
+        {
+            base.SystemDebugMessage(message);
+
+            WriteLine($"SystemDebugMessage: {message}");
+        }
+
+        /// <summary>
+        /// Set the current runtime statistics of the algorithm.
+        /// These are banner/title statistics which show at the top of the live trading results.
+        /// </summary>
+        /// <param name="key">Runtime headline statistic name</param>
+        /// <param name="value">Runtime headline statistic value</param>
+        public override void RuntimeStatistic(string key, string value)
+        {
+            base.RuntimeStatistic(key, value);
```

Comment:
I think this `override` seems unrequired?

Reply:
Thanks for catching this -- I meant to actually do something here :/ -- Will update!